lib/challenge.js

```diff
@@ -135,18 +157,27 @@ function prompt7() {
  */
 
 //  define someNumber here
-var someNumber;
 
-function prompt8() {
-  // print someNumber to the console in your function so you can see its value
-  console.log(someNumber);
 
+function prompt8() {
+  let someNumber = (Math.floor(Math.random() * 100));
+  
   // YOUR CODE HERE
+  if(someNumber < 30){
+    console.log("That's a small number you got there...");
+  }else if(someNumber < 60 && someNumber > 30){
+    console.log("The number is average");
+  }else{
+    console.log("This is really big!");
+  }
+  console.log(someNumber);
+  // print someNumber to the console in your function so you can see its value
 }
```

Comment:
Keep an eye on the spacing between the brackets and other characters.  Some people like to space between `if` and the first `(` for instance too.  I don't worry too much about that as long as you are consistent 

So it is easier to read I would rewrite it like:
```
 if (someNumber < 30){
    console.log("That's a small number you got there...");
  } else if (someNumber < 60 && someNumber > 30) {
    console.log("The number is average");
  } else {
    console.log("This is really big!");
  }
  console.log(someNumber);
  // print someNumber to the console in your function so you can see its value
}
```

Reply:
In what occasions would including a space not be good for?

Reply:
In most cases computers don't really care about the spaces, it is all about making the code readable to us humans. Each language has a general style guide to help the code be consistently formatted as to be readable from person to person. Keeping the formatting consistent just helps us all read this confusing stuff easier 😎